src/settings-ui/Settings.UI/SettingsXAML/Views/DashboardPage.xaml

```diff
@@ -408,6 +401,7 @@
                                 Background="Transparent"
                                 BorderThickness="0"
                                 Click="DashboardListItemClick"
+                                CornerRadius="{StaticResource OverlayCornerRadius}"
                                 Tag="{x:Bind Tag, Mode=OneWay}">
                                 <Grid
```

Comment:
Looks like this grid can be squashed with the child grid. It only has one child element.

Reply:
I’d argue making it into a control would be a better optimization

Reply:
I don't understand?

Reply:
The disable and enable can be made into a single control